src/liballoc/string.rs

```diff
@@ -268,7 +268,8 @@ use crate::vec::Vec;
 ///
 /// Here, there's no need to allocate more memory inside the loop.
 ///
-/// [`&str`]: str
+/// [`str`]: ../../std/primitive.str.html
+/// [`&str`]: ../../std/primitive.str.html
```

Comment:
Can you use `type@str`? Or will that run into #73423?

Reply:
In alloc, it points to remote link: https://doc.rust-lang.org/nightly/std/primitive.str.html
In std, the link is OK: http://0.0.0.0:8000/std/primitive.str.html